.prettierrc

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
?

Reply:
I moved prettier setting to `.eslintrc.js`.
I think it's more useful to be able to manage with one file.
What do you think?

https://github.com/react-hook-form/resolvers/blob/57795af37f395d4147ba5ee76c782f47afc247f5/.eslintrc.js#L15-L21

Reply:
personally, I prefer to keep them separate. They are two different technology, then others may ask why not keep `eslint` inside `prettier`.

Reply:
done.